mirdata/datasets/filosax.py

```diff
@@ -0,0 +1,510 @@
+"""
+
+INDEXES = {
+    "default": "full_1.0",
+    "full": "full_1.0",
+    "full_sax": "full_sax_1.0",
+    "lite": "lite_1.0",
+    "lite_sax": "lite_sax_1.0",
+    "test": "test",
+    "full_1.0": core.Index(filename="filosax_index_full_1.0.json"),
+    "full_sax_1.0": core.Index(filename="filosax_index_full_sax_1.0.json"),
+    "lite_1.0": core.Index(filename="filosax_index_lite_1.0.json"),
+    "lite_sax_1.0": core.Index(filename="filosax_index_lite_sax_1.0.json"),
+    "test": core.Index(filename="filosax_index_lite_1.0.json"),
+}
+
+DOWNLOAD_INFO = """
+To download the dataset, first go to the Zenodo pages below to request access:
+
+(Full - 14.5GB)
+https://zenodo.org/record/5643843#.YYL7aS2l3UI
+
+(Lite - 558MB)
+https://zenodo.org/record/5643734#.YYLQ-i2l3UI
+
+Unzip the downloaded file to the folder /Users/<username>/mir_datasets/ (or wherever data_home has been assigned on initialization), and remove the version number from the folder:
+
+(Full)
+/Users/<username>/mir_datasets/Filosax
+
+(Lite)
+/Users/<username>/mir_datasets/Filosax_Lite
+
+This data is sufficient to use the dataset in the "_sax" (sax only) mode. To download the backing data, go to the Aebersold sites:
+
+(Full)
+https://www.jazzbooks.com/mm5/merchant.mvc?&Screen=WISH&Store_Code=JAJAZZ&WishList_ID=1679
+
+(Lite)
+https://www.jazzbooks.com/mm5/merchant.mvc?&Screen=WISH&Store_Code=JAJAZZ&WishList_ID=1678
+
+Put the files downloaded into the "/Aebersold" folder, and then run the appropriate script from inside the home folder:
+
+(Full)
+python Scripts/Compile_Backing.py -version full
+
+(Lite)
+python Scripts/Compile_Backing.py -version lite
+
+which populates the "/Backing" folder with edited files, which match the versions that were used in the recordings.
+
+"""
+
+LICENSE_INFO = """
+The Filosax dataset contains copyright material and is shared with researchers under the following conditions:
+1. Filosax may only be used by the individual signing below and by members of the research group or organisation of this individual. This permission is not transferable.
+2. Filosax may be used only for non-commercial research purposes.
+3. Filosax (or data enabling the its reproduction) may not be sold, leased, published or distributed to any third party without written permission from the Filosax administrator.
+4. When research results obtained using Filosax are publicly released (in the form of reports, publications, or derivative software), clear indication of the use of Filosax shall be given, usually in the form of a citation of the following paper:
+    D. Foster and S. Dixon (2021),  Filosax: A Dataset of Annotated Jazz Saxophone Recordings.
+    22nd International Society for Music Information Retrieval Conference (ISMIR).
+5. Queen Mary University of London shall not be held liable for any errors in the content of Filosax nor damage arising from the use of Filosax.
+6. The Filosax administrator may update these conditions of use at any time. 
+"""
+
+
+class Note:
```

Comment:
this is not the ideal way of implementing this but I am fine with it. we have a an annotation class and a NoteData class inside annotations https://github.com/mir-dataset-loaders/mirdata/blob/master/mirdata/annotations.py 
ideally this should have been inherited Annotation. I suggest going on with this PR and opening another issue regarding this (which can be solved in the future).